projekt_do_pracy_na_labach/fiszki_backend/Dockerfile

```diff
@@ -1,2 +1,27 @@
+#Bartosz Habas
+
 FROM python:3.10
 
+WORKDIR /app
+
+COPY . /app
+RUN  pip install -r requirements.txt
+EXPOSE 8000 
+
+CMD ["python", "main.py"]
+
+
+
+#docker build --tag fiszki_backend .
+
+#docker run fiszki_backend 
+
+#docker run -p 8000 fiszki_backend 
```

Comment:
Brakuje Ci odniesienia skąd dokąd ma porty przekierować, podajesz to w ten sposób: `<port_udostępniany_przez_hosta>:<port_udostępniany_przez_kontener>`